packages/core/admin/ee/admin/hooks/useLicenseLimitsNotification/index.js

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
Isn't this a duplicate of `useLicenseLimitNotification` from above?

Reply:
Looks like it, i assume it was a bad commit issue, i'll sort it as gustav is OOO.